easybuild/easyconfigs/c/CMake/CMake-3.0.0-foss-2015a.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.0.0'
```

Comment:
I prefer keeping versions in sync across toolchains, this is a dep for netCDF 4.3.2, and we already have easyconfigs for this one